portfolio/src/main/webapp/index.html

```diff
@@ -11,34 +11,301 @@
   </head>
 
 
-  <body>
-    <section class="section">
-      <div class="container ">
-        <div class="notification is-info is-light">
-          <h1 class="title"> Bhargavi Kumar Sundaresan <i class="fas fa-hand-spock"></i></h1>
-          <p class="subtitle is-italic"> My portfolio!</p><br>
-          </div>
-        </div>
-
-        <div><br></div>
+  <body class="has-background-grey-lighter">
```

Comment:
Did you copy this code from somewhere? The indentation in index.html is messy, but it is fine for other files.

Reply:
Bulma has some examples. I just use them to fit my needs and continue writing inside it. I am working on indents and comments now. Will send it to you as soon as I am done! :)